### PR TITLE
Update NetworkPolicy content

### PR DESCRIPTION
**Pull request by jboxman**
- NetworkPolicy is supported by other CNI plug-ins
- Drop excessive usage of code face for NetworkPolicy

### CODE REVIEW

**Comment by jboxman** [comment]
@openshift/team-documentation, PTAL. Thanks!

File: modules/nw-networkpolicy-about.adoc [hunk 1]
```diff
@@ -6,33 +6,34 @@
 
 = About network policy
```
Comment (bergerhoffer) [inline modules/nw-networkpolicy-about.adoc]: @jboxman Just wondering if this should be "About NetworkPolicy"? That seems to be how you're referring to it everywhere else.

Comment (jboxman) [reply modules/nw-networkpolicy-about.adoc]: Yes, thanks!

**Comment by jboxman** [comment]
@vikram-redhat, should I close this and redo it against the new master once it's live?

**Comment by vikram-redhat** [comment]
@jboxman up to you. Whatever you feel comfortable with.

**Comment by jboxman** [comment]
Hi @zhaozhanqi, can you PTAL? I changed `NetworkPolicy` to NetworkPolicy in most places and updated the content in `modules/nw-networkpolicy-about.adoc` to clarify that other CNI plug-ins may support using NetworkPolicy too.

Thanks.

**Comment by jboxman** [comment]
@danwinship @squeed 

I updated the language to reflect that NetworkPolicy is not exclusive to OpenShift SDN. Let me know if that makes sense.

(Preview: http://file.bos.redhat.com/~jboxman/20190613/networkpolicy-naming/networking/configuring-networkpolicy.html)

Thanks.

**Comment by danwinship** [comment]
yes, looks right

**Pull request merged**